pre_push.py

```diff
@@ -0,0 +1,30 @@
+import shlex
```

Comment:
Also, I cannot tell from here, but can you ensure the file is executable? If that's not trivial from windows, I can set it later.